azure-core/azure/core/__init__.py

```diff
@@ -30,16 +30,18 @@
 from .configuration import Configuration
 from .pipeline.transport import HttpRequest
 from .exceptions import *
+from .credentials import SupportsGetToken
 
 
 __all__ = [
     "Configuration",
     "PipelineClient",
     "HttpResponseError",
+    "SupportsGetToken"
```

Comment:
Why do we have to include `SupportsGetToken` in __all__?

Reply:
We don't, just a mindless habit of mine, I'll remove it.